### PR TITLE
Rwa-1512-task search by role category CTSC

### PR DESCRIPTION
**Pull request by serdaroguzhanakin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-1512

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by sullivangareth — DISMISSED** [review]

**Review by Lee365 — DISMISSED** [review]
LGTM

**Review by BenNzewi — COMMENTED** [review]
What about the pact test scenario to show search request with roleCategory as param?

**Review by dharmendrak — APPROVED** [review]

File: src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagementGetTasksBySearchCriteriaConsumerTest.java [hunk 1]
```diff
@@ -433,4 +463,56 @@ private String createSearchEventCaseWithAvailableTasks() {
...
+                        .stringType("location_name", "Taylor House")
+                        .stringType("case_type_id", "Asylum")
+                        .stringType("case_id", "1617708245335311")
+                        .stringType("case_category", "Protection")
+                        .stringType("case_name", "Bob Smith")
+                        .booleanType("warnings", false)
+                        .stringType("case_management_category", "Some Case Management Category")
+                        .stringType("work_type_id", "hearing_work")
+                        .stringType("work_type_label", "Hearing work")
+                        .stringType("role_category", "CTSC")
```
Comment (BenNzewi) [inline src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagementGetTasksBySearchCriteriaConsumerTest.java:512]: Should this be stringValue?

Comment (serdaroguzhanakin) [reply src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagementGetTasksBySearchCriteriaConsumerTest.java]: changed

**Review by mkayad — APPROVED** [review]
LGTM

**Pull request merged**